pkgs/games/devilutionx/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit, cmake, gcc, SDL2, SDL2_mixer, SDL2_ttf, libsodium, pkg-config }:
+stdenv.mkDerivation rec {
+  version = "0.5.0-SNAPSHOT";
```

Comment:
I did not check the upstream project to see version info so apologies if this does not apply but...

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".

Reply:
`version = "unstable-2019-07-28";`

Reply:
Done